bittrex/test/bittrex_tests.py

```diff
@@ -18,6 +23,48 @@ def test_auth_basic_failures(unit_test, result, test_type):
     unit_test.assertIsNone(result['result'], "{0:s} failed response result not None".format(test_type))
 
 
+def mocked_buy_sell_withdraw_query(protection=None, path_dict=None, options=None):
+    return {"success": "true", "message": "", "result": {"uuid": "e606d53c-8d70-11e3-94b5-425861b86ab6"}}
+
+
+def mocked_cancel_query(protection=None, path_dict=None, options=None):
+    return {"success": "true", "message": "", "result": "null"}
+
+
+def mocked_get_order_query(protection=None, path_dict=None, options=None):
+    return json.loads(
+                        '{\
```

Comment:
Maybe import a multiline string from somewhere else or put it in a file.

Reply:
Ok, will change it to make the test file more readable

Reply:
Also for in the event the format changes we can just paste in what Bittrex gives us in the future.